### DISCUSSION TITLE
Custom window titles

### DISCUSSION THREAD

**Discussion by zjchen77** [lawand-dot-io/taskbar#123] [Feature requests]
Right now, chrome windows are previewed with the last tab's name. Showing windows as their own window names can help us to distinguish them better, thank you!

**Comment by lawand-dot-io** [comment]
Hello. I had this idea at one point. I will most likely add an option at some point which allows setting custom window titles for all apps, not just Chrome 👍

**Comment by lawand-dot-io** [comment]
Related: https://github.com/lawand-dot-io/taskbar/discussions/116
